boa/deployments.py

```diff
@@ -1,24 +1,36 @@
 import json
 import sqlite3
-from dataclasses import asdict, dataclass, fields
+import uuid
+from dataclasses import asdict, dataclass, field, fields
 from pathlib import Path
 from typing import Any, Optional
 
 from boa.util.abi import Address
 from boa.util.open_ctx import Open
 
+_session_id: str = None  # type: ignore
+
+
+def get_session_id():
+    global _session_id
+    if _session_id is None:
+        _session_id = str(uuid.uuid4())
+    return _session_id
+
 
 @dataclass(frozen=True)
 class Deployment:
     contract_address: Address  # receipt_dict["createAddress"]
-    name: str  # contract_name
+    contract_name: str
     rpc: str
     deployer: Address  # ostensibly equal to tx_dict["from"]
     tx_hash: str
-    broadcast_ts: float
+    broadcast_ts: float  # time the tx was broadcast
```

Comment:
```suggestion
    broadcast_time: datetime
```

Reply:
I think a float is fine. Datetime serializer/deserializer is not obvious, and I don't think we need it